Source/WebKit/UIProcess/WebFrameProxy.cpp

```diff
@@ -349,15 +349,10 @@ void WebFrameProxy::disconnect()
 
 void WebFrameProxy::didCreateSubframe(WebCore::FrameIdentifier frameID)
 {
-    // The DecidePolicyForNavigationActionSync IPC is synchronous and may therefore get processed before the DidCreateSubframe one.
-    // When this happens, decidePolicyForNavigationActionSync() calls didCreateSubframe() and we need to ignore the DidCreateSubframe
-    // IPC when it later gets processed.
-    if (WebFrameProxy::webFrame(frameID))
```

Comment:
We should replace with an assertion to make sure we were successful:
ASSERT(!WebFrameProxy::webFrame(frameID));

Reply:
This would be redundant with the MESSAGE_CHECK(m_process, WebFrameProxy::canCreateFrame(frameID));